### PR TITLE
fix webhooks handler setup and allow customize

### PR DESCRIPTION
**Pull request by ccarruitero**
This PR is sponsored by [MagmaLabs](https://www.magmalabs.io/)

### CODE REVIEW

**Comment by ccarruitero** [comment]
ping @aldesantis 

Error in linter is not related to the changes

**Comment by aldesantis** [comment]
@ccarruitero quick question on this: what's your use case for customizing the webhook handler? The handler already publishes events on the Solidus bus, meaning you can easily hook in your logic by listening to those events, so I'm curious to know what kind of customization you have that's not supported by this model.

**Comment by ccarruitero** [comment]
Well, in the project I have been working latest months, we have different carriers integrations (across easypost and other custom carriers apis). We have a custom shipment state machine and we don't use cartons. We need to track additional shipments states and preserve first ones, so instead to customize a carton we decide to use only shipments.

Current webhook handler relay on carton use to fire solidus events, but in our use case we are using only shipments. So need to use a custom handler

**Comment by aldesantis** [comment]
@ccarruitero okay, got it! Could you just update the documentation to reflect that the events will only be fired if you haven't customized your webhook handler?

**Comment by ccarruitero** [comment]
@aldesantis I updated docs. Let me know what you think

File: README.md [hunk 1]
```diff
@@ -119,6 +119,12 @@ Now, when Solidus gets a tracking update from EasyPost, a `solidus_easypost.trac
 will be fired. The event's payload will contain the `:carton` and `:payload` keys, with the
 `Spree::Carton` object associated to the tracker and the EasyPost payload respectively.
 
+### Customizing tracking webhook handler
```
Comment (aldesantis) [inline README.md]: This should be `Customizing the tracking webhook handler`. 🙏

File: README.md [hunk 2]
```diff
@@ -119,6 +119,12 @@ Now, when Solidus gets a tracking update from EasyPost, a `solidus_easypost.trac
...
+
+If you want to override default webhook handler, you can provide your own `webhook_handler_class` in your configuration.
+
+Note that if you override webhook handler, solidus events mentioned above don't be triggered unless your webhook handler class do
```
Comment (aldesantis) [inline README.md]: Can you change this slightly to the following?

```
Note that, if you override the webhook handler, no events will be fired out of the box.
```

**Review by aldesantis — CHANGES_REQUESTED** [review]
@ccarruitero just a couple of comments and then it should be good to merge, thanks!

**Comment by ccarruitero** [comment]
@aldesantis Thanks for your comments. I updated docs

**Review by aldesantis — APPROVED** [review]

**Comment by aldesantis** [comment]
@ccarruitero thanks for bearing with me on this one, great work!

**Pull request merged**